### PR TITLE
feature: mobile view for search component

### PR DESCRIPTION
**Pull request by cdxker**
mobile_ui

UI still needs fix for auto scroll to work properly. Will need to be fixed in another PR.

### CODE REVIEW

File: clients/search-component/src/utils/hooks/useChat.ts [hunk 1]
```diff
@@ -44,6 +44,14 @@ export const useChat = () => {
       if (doneReading) {
         done = doneReading;
         setIsLoading(false);
+
+        // Get First Chat Container 
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const chatContainer: any = modalRef.current?.querySelector("div.chat-container");
+        if (chatContainer) {
+          chatContainer.style.height = "fit-content";
+        }
```
Comment (cdxker) [inline clients/search-component/src/utils/hooks/useChat.ts:53]: Had to do a hack here to get the footer to be at the right position after a message is generated.

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
Spacing between user and assistant blocks is off. They also need a border drawn between them.

**Comment by cdxker** [comment]
image

Evened out the spacing.

**Comment by cdxker** [comment]
image

Added vertical bars

**Pull request merged**